R/pkg/R/mllib.R

```diff
@@ -69,6 +72,8 @@ NULL
 #'
 #' Makes predictions from a MLlib model. For more information, see the specific
 #' MLlib model below.
+#' @param object a fitted ML model object.
+#' @param ... additional argument(s) passed to the method.
```

Comment:
does it complain about this? This rd does not have a function signature so it shouldn't ask to document parameter?